src/utils/plotSpecs/generateHeatmapSpec.js

```diff
@@ -1,45 +1,45 @@
-const generateSpec = (config, groupName) => {
-  let legend = [];
-  if (config.legend.position === 'horizontal') {
-    legend = [
-      {
-        fill: 'color',
-        type: 'gradient',
-        orient: 'bottom',
-        direction: 'horizontal',
-        title: ['Intensity'],
-        labelFont: { value: config.fontStyle.font },
-        titleFont: { value: config.fontStyle.font },
-        gradientLength: {
-          signal: 'width',
-        },
+const generateSpec = (config) => {
+  let legend = [
+    {
+      fill: 'color',
+      type: 'gradient',
+      orient: 'bottom',
+      direction: 'horizontal',
+      title: ['Intensity'],
+      labelFont: { value: config.fontStyle.font },
+      titleFont: { value: config.fontStyle.font },
+      gradientLength: {
+        signal: 'width',
       },
-      {
-        fill: 'cellSetColors',
-        title: groupName,
-        type: 'symbol',
-        orient: 'top',
-        offset: 40,
-        symbolType: 'square',
-        symbolSize: { value: 200 },
-        encode: {
-          labels: {
-            update: {
-              text: { scale: 'cellSetIDToName', field: 'label' },
-            },
+    },
+    {
+      fill: 'cellSetColors',
+      title: 'Cluster ID',
```

Comment:
why hardcoding the title here?

Reply:
Because the leged shows the clusterIds that are shown (from metadata tracks)
![image](https://user-images.githubusercontent.com/22479584/111455551-9c8a2580-871e-11eb-90ed-f08986172290.png)

Reply:
It is always better to use a variable rather than hardocding that value. When you hardocde it like this, in the future if we wanted to display something else, not cluster id, we would have to change the config and adapt to it. Hardcoding values makes the code more fragile